static/js/reducers/email_test.js

```diff
@@ -0,0 +1,56 @@
+import configureTestStore from 'redux-asserts';
+
+import {
+  startEmailEdit,
+  updateEmailEdit,
+  clearEmailEdit,
+  START_EMAIL_EDIT,
+  UPDATE_EMAIL_EDIT,
+  CLEAR_EMAIL_EDIT,
+} from '../actions/email';
+import rootReducer from '../reducers';
+import { NEW_EMAIL_EDIT } from './email';
+import { assert } from 'chai';
+import sinon from 'sinon';
+
+describe('email reducers', () => {
+  let sandbox, store, dispatchThen;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+    store = configureTestStore(rootReducer);
+    dispatchThen = store.createDispatchThen(state => state.email);
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+
+    store = null;
+    dispatchThen = null;
+  });
+
+  let query = { query: "Hi, I'm a query!" };
+
+  let initialExpectation = { ...NEW_EMAIL_EDIT, ...{ query: query } };
```

Comment:
would this work?: `let initialExpectation = { ...NEW_EMAIL_EDIT, query: query };`